### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by eaglgenes101** [bheisler/TinyTemplate#16]
When compiling a project that uses `criterion`, which has `tinytemplate` as a dependency, I get this compilation error when cargo tries to build it: 
```
error: unknown `doc` attribute `Hidden`
  --> <SNIP>/tinytemplate-1.2.0/src/error.rs:44:11
   |
44 |     #[doc(Hidden)]
   |           ^^^^^^

error: aborting due to previous error

error: could not compile `tinytemplate`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```
My version of Cargo is `cargo 1.52.0-nightly (572e20153 2021-02-24)`.

**Comment by cart** [comment]
Yup this just broke Bevy's test builds https://github.com/bevyengine/bevy/pull/1313

**Comment by kitsuneninetails** [comment]
I also filed PR, only to see that a PR was already filed :).

**Comment by bheisler** [comment]
Thanks for the report! I've published a version 1.2.1 with the fix applied.
